LockwiseXCUITests/LockwiseXCUITests.swift

```diff
@@ -18,7 +18,30 @@ class LockwiseXCUITests: BaseTestCase {
         navigator.nowAt(Screen.WelcomeScreen)
     }
 
-    func testCheckEntryDetailsView() {
+    // To run this test locally, first run python3 upload_fake_passwordsBB.py 1
+    // See more info in docs/AutomatedTests.md
+    func testDeleteEntry() {
+        loginToEntryListView()
+        waitforExistence(app.tables.cells.staticTexts["aaafakeTesterDelete"])
+        // Need to add firstMatch for iPad case
+        app.tables.cells.staticTexts["aaafakeTesterDelete"].firstMatch.swipeLeft()
+        app.tables.buttons["Delete"].tap()
+        waitforExistence(app.alerts["Delete this login?"])
+        // First check the Cancel button
+        app.alerts.buttons["Cancel"].tap()
+        waitforExistence(app.tables.cells.staticTexts["aaafakeTesterDelete"])
+        app.tables.cells.staticTexts["aaafakeTesterDelete"].firstMatch.swipeLeft()
+        app.tables.buttons["Delete"].tap()
+        waitforExistence(app.alerts["Delete this login?"])
+        // Then Delete the login
+        app.alerts.buttons["Delete"].tap()
+        waitforExistence(app.navigationBars["firefoxLockwise.navigationBar"])
+        // Now check that the login has been removed
+        // Known issue #1074 on iPad, comment the verification until is fixed
```

Comment:
#1078 :)

Reply:
Nice! quick fix for the issue! if it gets merged before this one I will update this part. Thanks!